README.md

```diff
@@ -0,0 +1,31 @@
+## About Eclipse Che Workspace Data Sync
+
+This tooling provide ability to increase I/O performance for a developer workspaces.
+
+==============
+## Workspace lifecycle:
+
+#### Workspaces Startup phase:
```

Comment:
I propose the following format, but I'm not sure.

**Workspaces Startup phase:**
- Pods (workspace and data sync) are started in parallel

**Startup data sync phase:**
- Data flow goes from the persistent volume (rsync server) to the ephemeral volume (rsync client)

**Normal workspace usage phase:**
- Containers in the Workspace Pod have full R/W access to the ephemeral volume with restored project sources
- Container (rsync client) in the Workspace Pod have full R/W access to the persistent volume
- Data flow goes from rsync client to rsync server periodically by cron job

**Workspace Shutdown phase:**
- Containers in the Workspace Pod stopped and data flushed to the persistent volume

Reply:
Fixed